test/nri/plugin.go

```diff
@@ -209,6 +209,26 @@ func (p *plugin) RunPodSandbox(_ context.Context, pod *api.PodSandbox) error {
 	return nil
 }
 
+func (p *plugin) UpdatePodSandbox(_ context.Context, pod *api.PodSandbox) error {
```

Comment:
Eventually we should update this to something like below, so we can have more meaningful tests, but it should be perfectly fine to do that later with a subsequent PR.

```
func (p *plugin) UpdatePodSandbox(_ context.Context, pod *api.PodSandbox, overhead, resources *api.LinuxContainerResources) error {
    p.Lock()
    defer p.Unlock()

    // maybe look up pod and patch in the updates 

    // extra new overhead and resources also need to be added to the event struct type...
    p.emitEvent(
        &event{
            kind: "UpdatePodSandbox"
            pod: pod,
            overhead: overhead,
            resources: resources,
        },
    )

    return nil
}
```